### ISSUE TITLE
No Access to `DropDownMenu` or `AlertDialog` in common source set

### ISSUE THREAD

**Issue by ScottPierce** [JetBrains/compose-multiplatform#762]
We currently can't access the `DropDownMenu` or `AlertDialog` from the common source set for compose. Someone mentioned this was because of `PopupProperties`.

**Comment by igordmn** [comment]
For `DropDownMenu` we can create a common function in `commonMain`:

```
fun DropdownMenu(
    expanded: Boolean,
    onDismissRequest: () -> Unit,
    modifier: Modifier = Modifier,
    offset: DpOffset = DpOffset(0.dp, 0.dp),
    content: @Composable ColumnScope.() -> Unit
)
```

**Comment by avently** [comment]
@igordmn I did it and here is the result
https://github.com/JetBrains/compose-jb/issues/758

**Comment by atsushieno** [comment]
What is blocking you from adding DropdownMenu as an `expected` declaration?

**Comment by francismariano** [comment]
Is there some milestone for this issue??
Access AlertDialog and DropDownMenu from common sources set sounds great.

compose-android // compose-ios // compose-desktop // compose-web

**Comment by MatkovIvan** [comment]
It was released in 1.5.0

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6040">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
